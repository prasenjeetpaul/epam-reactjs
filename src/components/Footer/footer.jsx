import React from 'react';
import './footer.scss';

export const Footer = () => (
    <footer>
        <span className="logo">
            <b>netflix</b>roulette
        </span>
    </footer>
);
