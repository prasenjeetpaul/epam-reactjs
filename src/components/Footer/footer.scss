@import "../../assets/style/variables.scss";
@import "../../assets/style/util.scss";

footer {
    width: 100%;
    height: 50px;
    flex-shrink: 0;
    display: flex;
    justify-content: center;
    align-items: center;
    background-color: $grey;
}
