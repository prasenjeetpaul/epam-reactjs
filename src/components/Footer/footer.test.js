import React from 'react'
import { render } from '@testing-library/react';
import { Footer } from './footer';

describe('Footer', () => {
    it('should match the snapshot after render', () => {
        const { asFragment } = render(<Footer />);
        expect(asFragment()).toMatchSnapshot();
    });
});