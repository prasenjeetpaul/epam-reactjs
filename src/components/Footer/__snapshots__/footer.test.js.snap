// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot after render 1`] = `
<DocumentFragment>
  <footer>
    <span
      class="logo"
    >
      <b>
        netflix
      </b>
      roulette
    </span>
  </footer>
</DocumentFragment>
`;
